app/src/debug/java/net.squanchy/DebugActivity.java

```diff
@@ -63,9 +64,9 @@ private void createAndNotifyTalksCount(int count) {
     private Event createTestEvent(int id) {
         Calendar calendar = Calendar.getInstance();
         calendar.add(Calendar.MINUTE, 5);
-        Date start = calendar.getTime();
+        DateTime start = new DateTime(calendar.getTime());
```

Comment:
❌ don't use Calendar (Java's own `Date` APIs) mixed with JodaTime's `DateTime`.

Here you should have:

```java
DateTime start = new DateTime().plusMinutes(5);
DateTime end = start.plusMinutes(45);
```